notebooks/116-sparsity-optimization/116-sparsity-optimization.ipynb

```diff
@@ -8,7 +8,9 @@
    },
```

Comment:
<p>could you pleae also add links on BERT model (paper possibly or model card) and SST-2 dataset?</p><p> </p><p>&gt;&gt; Download and quantize sparse BERT model from a public source using the OpenVINO integration with Hugging Face Optimum.</p><p>here no quantization step, just download already sparse model... Is it expected?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/903/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
@AlexKoff88  is it expected?
Maybe it is better to split tutorial on 2 notebooks:
1. quantize model (in case if somebody interested in reproducing results)
2. run optimized model inference

Reply:
yes, this is expected. The optimization recipe can be found in the corresponding model card (it's worth mentioning in the comments). The main goal is to demonstrate the inference capabilities while optimization itself requires fine-tuning and different HW. We will mention extra steps in the blog we are planning.

Reply:
@eaidova Thanks for reviewing, links to model card, sst2 are added.